CodingTracker/Validator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Globalization;
+namespace CodingTracker
+{
+    internal static class Validator
+    {
+        public static bool IsValidFilterOption(string? input)
+        {
+            string[] validOptions = { "a", "d", "w", "m", "y" };
+            foreach (string validOption in validOptions)
+            {
+                if (input == validOption)
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        public static bool IsValidOption(string? input)
+        {
+            string[] validOptions = { "v", "a", "d", "u", "srt", "stp", "0" };
+            foreach (string validOption in validOptions)
+            {
+                if (input == validOption)
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        public static bool IsValidDateInput(string input) => DateTime.TryParseExact(input, "dd-MM-yy HH-mm-ss", new CultureInfo("en-US"), DateTimeStyles.None, out _);
```

Comment:
![image](https://user-images.githubusercontent.com/108711476/204121900-3c2149c9-db67-4806-ba12-3387bc00f546.png)

It looks like I'm being rejected here because I'm in Australia. How would you fix this? It's a good problem to solve. Look into invariant culture.